.github/workflows/codecov.yml

```diff
@@ -11,15 +11,19 @@ jobs:
     env:
       PYTHON: '3.9'
```

Comment:
Need this? Inconsistent with py 3.11 further down

Reply:
Thanks, missed that. Will remove!